monitor/measure.go

```diff
@@ -185,6 +189,59 @@ func (cm *CounterIOMeasure) Record() {
 	cm.baseTx = bTx
 }
 
+// CounterDockerMeasure is a struct that takes a docker-id and measures
+// system and user time.
+type DockerMeasure struct {
+	name   string
+	docker []string
+	sys    []float64
+	user   []float64
+}
+
+// NewDockerMeasure returns an CounterIOMeasure fresh. The base value are set
+// to the current value of counter.Rx() and counter.Tx()
+func NewDockerMeasure(name string, docker []string) *DockerMeasure {
+	return &DockerMeasure{
+		name:   name,
+		docker: docker,
+	}
+}
+
+func (dm *DockerMeasure) Update() {
+	if len(dm.docker) == 0 {
+		// Search all docker images
+	}
+	for i, d := range dm.docker {
+		cpuFile := "/sys/fs/cgroup/cpuacct/docker/" + d + "/cpuacct.stat"
```

Comment:
Not necessarily in the comment but could you share your knowledge of this ? What's this file etc ?  I'm curious about that..
*Edit*: ok no worries I've found this quite useful : https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Resource_Management_Guide/ch01.html